scripts/test-make-translations.sh

```diff
@@ -6,6 +6,11 @@ source ./scripts/bash-utils.sh
 
 log_group_begin "Check: translations"
 
+pip install --upgrade pip
+# install requirements, excluding difficult to install ones
+grep -Ev 'xmlsec|python3-saml' requirements/dev-requirements.txt > ../requirements.txt
+pip install -r ../requirements.txt
```

Comment:
These extra `pip install`s will add more time to every test run. The docker build optimizes the installation of requirements by [pre-installing them in the docker image](https://github.com/dimagi/commcare-hq/blob/323e6e65a41ae1801604646e723bfa78f48004d3/Dockerfile#L46) to minimize the overhead of [`pip-sync` during test container initialization](https://github.com/dimagi/commcare-hq/blob/323e6e65a41ae1801604646e723bfa78f48004d3/docker/run.sh#L34).

What do you think of moving whatever is needed here from dev-requirements.in to test-requirements.in so they are installed during docker setup?

Reply:
I think this no longer applies because the extra pip installs have been removed in a subsequent commit. Feel free to ignore/resolve if so.